src/irr/asset/CGLSLBRDFBuiltinIncludeLoader.h

```diff
@@ -61,7 +77,7 @@ struct irr_glsl_BSDFIsotropicParams
    // basically metadata
    vec3 L;
    float invlenL2;
```

Comment:
I don't think anybody needs these anymore

Reply:
same for `T,B` in `irr_glsl_BSDFAnisotropicParams` (which also needs to use `irr_glsl_AnisotropicViewSurfaceInteraction` btw)

Reply:
let me know if tackled in new implementation.

Reply:
yea, i separated them, T and B are in aniso interaction now